registration/templates/base/base.html

```diff
@@ -11,13 +11,17 @@
 
     <!-- Bootstrap core CSS -->
     <link href="{%  static 'base/css/bootstrap.min.css' %}" rel="stylesheet">
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
```

Comment:
Can't accept changes to base.html, It will affect all other page. You can use {%block css%} your css {% endblock %}, Add changes only to the profile.html